src/popup/components/TransactionList.vue

```diff
@@ -203,7 +205,11 @@ export default defineComponent({
       }
     }
 
-    function checkLoadMore() {
+    async function checkLoadMore() {
+      // sleep is required because viewportElement scrollHeight
+      // is not being changed fast enough, after filter is applied
+      await sleep(50);
```

Comment:
Maybe using `await nextTick()` would solve the issue and better fit into the Vue/DOM environment?